src/client/routes/dashboard/index.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect, Dispatch } from 'react-redux';
+import { IAppState } from '../../redux/reducers';
+import { actions } from './reducer';
+
+const Dashboard = () => {
+  return <div>Dashboard page</div>;
+};
+
+const mapStateToProps = (data: IAppState) => ({ data });
```

Comment:
`data` should be state

Reply:
const mapStateToProps = ({dashboard}) => ({ dashboard });
As we are only need to bind the dashboard state here.